stock-usecase/src/main/java/org/dhis2/android/rtsm/ui/home/HomeViewModel.kt

```diff
@@ -45,7 +50,10 @@ class HomeViewModel @Inject constructor(
     savedState: SavedStateHandle,
 ) : BaseViewModel(schedulerProvider) {
 
-    private val config: AppConfig = savedState.get<AppConfig>(INTENT_EXTRA_APP_CONFIG)
+    private val _config = MutableStateFlow<StockUseCase?>(null)
+    private val config: StateFlow<StockUseCase?> = _config
```

Comment:
config is immutable right?, what do you think about having just one property of type StockUseCase like: `private lateinit var config: StockUSeCase`  and init it in the loadstockUseCase() method that you created?